src/CodeGen_RISCV.cpp

```diff
@@ -21,11 +21,28 @@ string CodeGen_RISCV::mcpu() const {
 }
 
 string CodeGen_RISCV::mattrs() const {
-    return "";
+    string arch_flags = "+m,+a,+f,+d,+c";
```

Comment:
There is a logic to choose a default march in Clang & GCC,as described in [1].
When it is not explicitly specified, it is "rv32|64imafdc", which supports standard extensions:
- Integer Multiplication and Division,
- Atomic Instructions,
- Single-Precision Floating-Point, 
- Double-Precision Floating-Point,
- Compressed Instructions.

so I changed it to follow the convention.
[1] https://github.com/llvm/llvm-project/blob/main/clang/lib/Driver/ToolChains/Arch/RISCV.cpp#L721

Reply:
Ah, cool. It would probably be good to capture that in a comment in this function, since few readers will immediately know what those flags mean without reference.

Reply:
no problem!